CHANGELOG-2.7.md

```diff
@@ -9,6 +9,7 @@ To get the diff between two versions, go to https://github.com/symfony/symfony/c
 
 * 2.7.0-BETA1 (2015-04-10)
 
+ * feature #14434 [Translation] add options 'as_tree', 'inline' to YamlFileDumper (gepo)
```

Comment:
you should update [CHANGELOG of Translation](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Translation/CHANGELOG.md) and the target should be `2.8`